recipes/ckan-setup.rb

```diff
@@ -24,11 +24,23 @@
 	package p
 end
 
+# Install packages that have different names on different systems
+node['datashades']['ckan_web']['alternative_packages'].each do |p|
+	bash "Install #{p} if available" do
+		code <<-EOS
+			if (yum info "#{p}"); then
+				yum install -y "#{p}"
+			fi
+		EOS
+	end
+end
+
 # Create CKAN Group
 #
 group "ckan" do
 	action :create
-	gid '1000'
+	gid '2000'
```

Comment:
does this handle existing systems or it won't run?

Reply:
It handles existing systems by checking whether `/home/ckan` already exists, line 43.

Reply:
hmm, if we re-create a new instance on linux1, would it still work? this should really be checking its own variables not whats on disk which may not be there just yet in how chef/puppet works.